Application/Frontend/components/FriendsColumn/FriendsTab.tsx

```diff
@@ -88,7 +88,7 @@ export function FriendsTab(props: TextInputProps) {
 
     if (activeChat && user?.id) { // Ensure both activeChat and user.id are defined
       return <Chat 
-          sender={user.id}
+          sender={user.username as any}
```

Comment:
Great change to using username so that it fits with our current database field name